docs/version-control: Set commit message line limit to 70.

chrisbobbe opened the pull request:
@gnprice has commented that commit message lines should wrap at
about 68 columns, but not exceed 70.

https://github.com/zulip/zulip-mobile/pull/3404#issuecomment-477411875

timabbott commented:
I started a discussion in https://chat.zulip.org/#narrow/stream/3-backend/topic/commit.20line.20length/near/784952.

timabbott commented:
@chrisbobbe thanks for bringing this to our attention.  The discussion on the chat.zulip.org thread (linked above) seems to have terminated; if you'd like to update this PR to reflect the discussion there, that'd be great!  Or if you're not sure how to explain it, let me know and I'll do the tweaks and integrate this.

chrisbobbe commented:
Sure! I've just updated the pull request, drawing from Greg's comment that I linked to, above. I'm afraid I won't be available for about a week or so starting this evening, so if you have any changes you'd like to make, please go ahead. In fact, yes -- I see I've introduced some trailing whitespace; I'm sorry about that. Also, for the future, would you prefer that I squash multiple commits in a pull request, or keep them separate to maintain a granular history? Thank you!

timabbott commented:
Generally for this sort of thing squashing is correct, since we really only want to apply one change to master.  Check out the Zulip commit guidelines for more details: https://zulip.readthedocs.io/en/latest/contributing/version-control.html

timabbott commented:
Merged as 04e6c3bd18e17fbfb1f59e6aedc060fa7f82e957, after some cleanup to the formatting.  Thanks @chrisbobbe!
